textproc/docbook-xsl/Portfile

```diff
@@ -58,6 +60,9 @@ destroot {
     foreach pathname ${srcdocpaths} {
         copy ${worksrcpath}/${pathname} ${destroot}${prefix}/${instdocdir}
     }
+
+    copy ${destroot}${prefix}/${instxsldir}/catalog.xml ${destroot}${prefix}/${instxsldir}/catalog.sf.xml
```

Comment:
Suggestion: I would have added a short comment here to explain why this is being done, it's possible that somebody who's looking at this months from now could have a hard time understanding the reasoning.

Reply:
Done.